### ISSUE TITLE
Wasm3 crash on Windows 10

### ISSUE THREAD

**Issue by secana** [wasm3/wasm3#135]
Hi, I ran into problems with wasm3 on Windows.

If I try to run wasm3 0.4.7 on Windows 10 I get the following error message:

```powershell
❯ wapm run wasm3
Usage:
  wasm3 <file> [args...]
  wasm3 --func <function> <file> [args...]
  wasm3 --repl [file]
thread 'main' panicked at 'not yet implemented: windows::trigger_trap', lib\clif-backend\src\signal\windows.rs:124:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
Error: error: "unhandled trap at 7ffda1aa3e49 - code #e06d7363: unknown exception code"
```

If I try to execute only a function, I get the following error message:

```powershell
wapm run wasm3 --func add .\wasm-files\add-module.wasm 1 2
Error: [Fatal] repl_load: cannot open file
Error: cannot open file ()
```

Any ideas how to get wasm3 to run on Windows?

**Comment by vshymanskyy** [comment]
This is WAPM issue,not wasm3.
Tru executing wasm3 directly

**Comment by secana** [comment]
Sorry, maybe a stupid question, but how do I invoke wasm3 directly? The binary is not in any path and as such cannot be found.

**Comment by vshymanskyy** [comment]
@secana you can try downloading the prebuild binary here: https://github.com/wasm3/wasm3/releases/latest
Or you may try building one yourself (if the above doesn't work).
For simplicity, put in in the same directory as your wasm file, and run `wasm3` command from the same directory.

**Comment by vshymanskyy** [comment]
Closed this ticket, but please continue the discussion if needed

**Comment by vshymanskyy** [comment]
Did it work?

**Comment by secana** [comment]
Sorry for the late answer. Yes it works, thx for the help.
To prevent others from running into the same issue, it maybe a change on the "Getting Started" page would be useful: https://wapm.io/package/vshymanskyy/wasm3

Currently the way shown their is the installation with wpm. A link to the prebuild binaries would help a lot.

**Comment by felixroos** [comment]
Also just ran into this issue on macOS. Installed wasm3 from wapm via `wapm install wasm3 -g`.
Running `wasm3 anyfile.wasm` fails with:

```sh
Error: [Fatal] repl_load: cannot open file
Error: cannot open file ()
```

After finding this issue, I built wasm3 from the source and it worked. It would be really helpful to have some hint on that in the readme or on the package page, especially for wasm noobs like me (this was the first time I touched a wasm file). Anyways thank you for building such a cool tool
